common/COMMON_moveReportToRSlims.py

```diff
@@ -1,59 +1,65 @@
-	for o,p in opts:
-		if o == '-b':
-			args[ "BASEURI" ] = p
-		elif o == '-u':
-			args[ "USERNAME" ] = p
-		elif o == '-p':
-			args[ "PASSWORD" ] = p
-		elif o == '-l':
-			args[ "processID" ] = p
-
-	BASEURI = args["BASEURI"].split('api')[0]
-
-	lims = Lims(BASEURI, args["USERNAME"], args["PASSWORD"])
-	lims.check_version()
-
-	process = Process(lims, id= args[ "processID" ])
-
-	# for each output artifact, download the attached PDF and move to the RS LIMS file surface
-	for inArt, outArt in process.input_output_maps:
-		if outArt['output-generation-type'] == 'PerInput':
-			if not outArt['uri'].samples[0].id.startswith("SAL1A"): # do not move files for test-samples
-				# get the name that the file has in Clarity LIMS
-				fileName = outArt['uri'].files[0].original_location.split('/')[-1]
-				# create the path to the RS LIMS file surface
-				path = "/mnt/limsrs-clarity/PDF/in/" + fileName
-				# get the file as a readcontent object
-				r = lims.get_file_contents(uri = outArt['uri'].files[0].uri)
-
-				# write as binary to the specified path 
-				with open(path, 'wb') as f:
-					f.write(r.read())
-
-				print("Report moved to RS Lims, %s" % fileName)
-
-			else:
-				print("Test sample, report not moved, %s" % outArt['uri'].name)
-
-if __name__=='__main__':
-	main()
+    args = setUpArgs()
+    set_logger_log_start_script(args["processID"], __file__, args["USERNAME"])
+    lims = setUpLIMS(args)
+
+    process = Process(lims, id=args["processID"])
+    inart_actions = process.step.actions.next_actions
+    outarts = process.all_outputs()
+    # For each output artifact, download the attached PDF and move to the RS LIMS file surface.
+    for action in inart_actions:
+        inart = action["artifact"]
+        # Only move report if sample is marked as complete
+        if not "complete" in action["action"]:
+            logging.info(f"Sample not complete, report not moved, {inart.name}")
+        else:
+            outart = get_outart(outarts, inart)
+            if not outart.files:
+                logging.info(f"No report for {outart.name}")
+            else:
+                # Get the name of the file in Clarity LIMS
+                filename = outart.files[0].original_location.split("/")[-1]
+                # Create the path to the RS LIMS folder
+                path = RS_LIMS_PAHT + filename
+                # Get the file as a readcontent object
+                read_content = lims.get_file_contents(uri=outart.files[0].uri)
+
+                project = outart.samples[0].project
+                if project.udf.get("Project Classification", "") == "Laborativ - Klinisk":
+                    # Write binary report
+                    with open(path, "wb") as f:
+                        f.write(read_content.read())
+                    logging.info(f"Report moved to RS Lims, {filename}")
+                else:
+                    logging.info(f"Test sample, report not moved, {outart.name}")
```

Comment:
```suggestion
            if not outart.files:
                logging.info(f"No report for {outart.name}")
                continue

            # Get the name of the file in Clarity LIMS
            filename = outart.files[0].original_location.split("/")[-1]
            # Create the path to the RS LIMS folder
            path = RS_LIMS_PAHT + filename
            # Get the file as a readcontent object
            read_content = lims.get_file_contents(uri=outart.files[0].uri)

            project = outart.samples[0].project
            if project.udf.get("Project Classification", "") == "Laborativ - Klinisk":
                # Write binary report
                with open(path, "wb") as f:
                    f.write(read_content.read())
                logging.info(f"Report moved to RS Lims, {filename}")
            else:
                logging.info(f"Test sample, report not moved, {outart.name}")
```

You can just add a `continue` after L43  to advance the for loop and drop the first else-block.